response_operations_ui/views/messages.py

```diff
@@ -53,8 +56,8 @@
 @login_required
 def create_message():
     if not user_has_permission("messages.edit"):
-        logger.exception("No message edit role")
-        abort(401)
+        logger.error("No message edit role")
```

Comment:
Without any biddings (i.e user_id) and the volume of log message we get do these add anything? With it raising to NoPermissionError which also has an error do we need them?

Reply:
Agreed.  Just changed it so we add the missing permission to the exception, and the NoPermissionError handler logs both the users id and the missing permission